connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/PostgresIntegrationSuite.scala

```diff
@@ -138,6 +139,12 @@ class PostgresIntegrationSuite extends DockerJDBCIntegrationSuite {
       "c0 money)").executeUpdate()
     conn.prepareStatement("INSERT INTO money_types VALUES " +
       "('$1,000.00')").executeUpdate()
+
+    conn.prepareStatement(s"CREATE TABLE timestamp_ntz(v timestamp)").executeUpdate()
```

Comment:
can we also add a column with `timestamp without time zone`?

Reply:
It seems Postgres have timestamp without time zone. I think we can mapping timestamp -> timestamp, timestampNTZ -> timestampNTZ.
H2 does not have timestampNTZ.

Reply:
isn't it the default of `timestamp`?

Reply:
Yes, in Postgres `timestamp` is equivalent to timestamp without time zone.

Reply:
It is actually the same case with H2 as well. `timestamp == timestamp without timezone` http://www.h2database.com/html/datatypes.html#timestamp_type